pkg/cmd/bundle/push.go

```diff
@@ -80,6 +83,7 @@ Input:
 		},
 	}
 	c.Flags().StringSliceVarP(&opts.bundleContentPaths, "filenames", "f", []string{}, "List of fully-qualified file paths containing YAML or JSON defined Tekton objects to include in this bundle")
+	c.Flags().StringSliceVarP(&opts.annotationParams, "annotate", "a", []string{}, "OCI Manifest annotations in the form of key=value to be added to the OCI image")
```

Comment:
`-a` flag sounds bit odd 😅 , let's keep `--annotate` only and not add any short-hand of this flag.
@piyush-garg @chmouel @vdemeester please let me know your thoughts on it